cluster-autoscaler/cloudprovider/azure/azure_scale_set_test.go

```diff
@@ -427,8 +436,10 @@ func TestScaleSetNodes(t *testing.T) {
 	assert.NotNil(t, ss)
 	instances, err := group.Nodes()
 	assert.NoError(t, err)
-	assert.Equal(t, len(instances), 1)
```

Comment:
This test case had VMSS capacity as 3 and VMSS VMs as 1.

Updated to make both consistent as 3.